dracut/fetch-driver-net.sh

```diff
@@ -5,7 +5,7 @@
 command -v warn_critical >/dev/null || . /lib/anaconda-lib.sh
 
 # initqueue/online hook passes interface name as $1
-netif="$1"
+#netif="$1"
```

Comment:
Why is this commented out?

Reply:
Sorry, that's a mistake. The variable was not used. Actually, do we want to keep the comment above or drop it too?

Reply:
I would rather remove it. For these situations there is git history.